packages/web/src/components/spa/HackAUBG/HackAUBG.jsx

```diff
@@ -31,42 +30,61 @@ export const HackAUBG = () => {
         new Anchor('FAQ', '#faq', <FaRegLightbulb />),
     ];
 
+    const match = useMatch('/hackaubg/verify');
+
+    const [showVerification, setShowVerification] = useState(!!match);
+
+    useEffect(() => {
+        document.body.style.overflow = showVerification ? 'hidden' : 'auto';
+    }, [showVerification]);
+
+    const handleVerificationSuccess = () => {
+        setShowVerification(false);
+    };
+
     return (
-        <div className="hackaubg-container">
-            <NavBar
-                props={
-                    new Props(
-                        anchorList, // list of anchors
-                        false, // hackAUBG button
-                        'rgba(0,0,0,.5)', // desktop background color nav
-                        true, // sticky desktop nav
-                        '#222222', // mobile nav background color when not opened (default transparent)
-                        'gray', // mobile background color nav when opened
-                        '#e2d7fc', // anchor color
-                        'white', // desktop anchor hover color
-                        'dark gray', // mobile anchor hover color
-                    )
-                }
-            />
-            <LandingPage />
-            <AboutHackathon />
-            <JourneySection />
-            <RegistrationForm />
-            <MentorsSection />
-            <JudgesSection />
-            <VideoSection />
-            <ScheduleHackathon />
-            <GradingCriteria />
-            <AwardsSection />
-            <Sponsors />
-            <FaqSection />
-            <Footer
-                color={'rgba(220,193,255,255)'}
-                iconColor={'rgb(0, 0, 0)'}
-                iconBgColor={'rgba(0, 0, 0, 0)'}
-                textColor={'#000'}
-                iconSize={'2.6em'}
-            />
+        <div className={`hackaubg-container ${showVerification ? 'showVerification' : ''}`}>
```

Comment:
Why the two different class names here?

Reply:
So that styles can be conditionally applied based on the showVerification state.